include/seqan3/alphabet/nucleotide/nucleotide_base.hpp

```diff
@@ -0,0 +1,160 @@
+ * \tparam char_t       The character type of the alphabet (set this to `void` when defining just a
+ *                      seqan3::semi_alphabet_concept).
+ */
+template <typename derived_type, auto size, typename char_t = char>
+class nucleotide_base : public alphabet_base<derived_type, size, char_t>
+{
+private:
+    //!\brief Type of the base class.
+    using base_t = alphabet_base<derived_type, size, char_t>;
+
+    /*!\name Constructors, destructor and assignment
+     * \{
+     */
+    constexpr nucleotide_base() : base_t{} {}
+    constexpr nucleotide_base(nucleotide_base const &) = default;
+    constexpr nucleotide_base(nucleotide_base &&) = default;
+    constexpr nucleotide_base & operator=(nucleotide_base const &) = default;
+    constexpr nucleotide_base & operator=(nucleotide_base &&) = default;
+    //!\}
+
+    //! Befriend the derived_type so it can instantiate.
+    friend derived_type;
+
+public:
+    // Import from base:
+    using typename base_t::char_type;
+    using typename base_t::rank_type;
+    using base_t::value_size;
+    using base_t::to_rank;
+
+    /*!\name Letter values
+     * \brief Static member "letters" that can be assigned to the alphabet or used in aggregate initialization.
+     * \details Similar to an Enum interface.
+     */
+    //!\{
+    static derived_type constexpr A       = assign_char(derived_type{}, 'A');
+    static derived_type constexpr B       = assign_char(derived_type{}, 'B');
+    static derived_type constexpr C       = assign_char(derived_type{}, 'C');
+    static derived_type constexpr D       = assign_char(derived_type{}, 'D');
+    static derived_type constexpr G       = assign_char(derived_type{}, 'G');
+    static derived_type constexpr H       = assign_char(derived_type{}, 'H');
+    static derived_type constexpr K       = assign_char(derived_type{}, 'K');
+    static derived_type constexpr M       = assign_char(derived_type{}, 'M');
+    static derived_type constexpr N       = assign_char(derived_type{}, 'N');
+    static derived_type constexpr R       = assign_char(derived_type{}, 'R');
+    static derived_type constexpr S       = assign_char(derived_type{}, 'S');
+    static derived_type constexpr T       = assign_char(derived_type{}, 'T');
+    static derived_type constexpr U       = assign_char(derived_type{}, 'U');
+    static derived_type constexpr V       = assign_char(derived_type{}, 'V');
+    static derived_type constexpr W       = assign_char(derived_type{}, 'W');
+    static derived_type constexpr Y       = assign_char(derived_type{}, 'Y');
+    static derived_type constexpr UNKNOWN = assign_char(derived_type{}, 'N');
+    //!\}
+
+    /*!\name Conversion operators
+     * \{
+     */
+//     //!\brief Implicit conversion between dna* and rna* of the same size.
+//     //!\tparam other_nucl_type The type to convert to; must satisfy seqan3::nucleotide_concept and have the same \link value_size \endlink.
+//     template <typename other_nucl_type>
```

Comment:
If this is part of a follow up issue reference the issue in a comment

Reply:
No, this can be removed!